vsphere/resource_vsphere_host.go

```diff
@@ -114,22 +114,20 @@ func resourceVsphereHostRead(d *schema.ResourceData, meta interface{}) error {
 			d.SetId("")
 			return nil
 		}
-		log.Printf("[DEBUG] Error while searching host %s. Error: %s", hostID, err)
-		return err
+		return fmt.Errorf("error while searching host %s. Error: %s ", hostID, err)
```

Comment:
Is it worth pulling the string "error while searching ...." into a local variable? Seems to be repeated a lot

Reply:
Seems like an overkill, I'd rather come up with original error messages :)